app/views/docs/databases-relationships.phtml

```diff
@@ -0,0 +1,137 @@
+
+<h3>Tradeoff</h3>
+<p>Consider using relationships when the same information is found in multiple places to avoid duplicates. However, relationships come with the tradeoff of slowing down queries. For applications where the best read and write performance is important, it may be acceptable to tolerate duplicate data.</p>
+
+<h2><a href="/docs/databases-relations#configurable-options" id="configurable-options">Configurable Options</a></h2>
+
+<h3><a href="/docs/databases-relations#directionality" id="directionality">Directionality</a></h3>
+<p>Appwrite relationships can be one-way or two-way.</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 250px">Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>One-way</code></td>
+            <td>The relationship is only visible to one side of the relation. For example, subscribers on YouTube is one-way. If you subscribe to a channel, they don't automatically subscribe back.</td>
+        </tr>
+        <tr>
+            <td><code>Two-way</code></td>
+            <td>The relationship is visible to both sides of the realtion. For example, friends on social media is two-way. If you're someone's friend, they're automatically your friend as well.</td>
+        </tr>
+    </tbody>
+</table>
+
+<h3><a href="/docs/databases-relations#cardinality" id="cardinality">Cardinality</a></h3>
+<p>Appwrite provides four different relationship types to enforce different associative rules between documents.</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 250px">Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>One-to-one</code></td>
+            <td>A document can only be related with one and <b>only</b> one document. If you try to relate multiple documents in a one-to-one relationship, Appwrite throws an error. For example, each user has only one profile.</td>
+        </tr>
+        <tr>
+            <td><code>One-to-many</code></td>
+            <td>A document can be related to many other documents. For example, a user can create many posts.</td>
+        </tr>
+        <tr>
+            <td><code>Many-to-one</code></td>
+            <td>Many documents can be related to a single document. For example, many posts can share the same author.</td>
+        </tr>
+        <tr>
+            <td><code>Many-to-many</code></td>
+            <td>A document can be related to many other documents. For example, a user can have many friends and many users can share the same friend.</td>
+        </tr>
+    </tbody>
+</table>
```

Comment:
These explanations are okay. It just always takes me a while to really think about it to understand it...and then I always have to review it again the next time.

Reply:
🤔 I've yet to see a better explanation. The way I learned Cardinality in entity-relationships was way more complicated...

I wish I could make this easier to understand.